### ISSUE TITLE
State of MS Word first line paragraph indent toggle is not announced

### ISSUE THREAD

**Issue by XLTechie** [nvaccess/nvda#16189]
### Steps to reproduce:

* Create a new document in MS Word.
* Press Ctrl+0.
* Press NVDA+f.
* Press Ctrl+0 again
* Press NVDA+f again

### Actual behavior:

Nothing is announced when the key (toggle) is pressed, so there is no way to know the state has changed.
Pressing NVDA+f, also does not reveal the state of this toggle, even though it does mention line spacing.

### Expected behavior:

The result of the toggle should be announced when pressed, like the subscript toggle (Ctrl+=).

NVDA+f should probably announce the state of this toggle if paragraph spacing is enabled as well.

#### NVDA installed/portable/running from source:

Installed or portable

#### NVDA version:

alpha-31030,421fe4ef

#### Windows version:

11 home 23H2

#### Name and version of other software in use when reproducing the issue:

Microsoft Office 16.0.17231.20194 (Office 2019)

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

Several subversions of 2023.X and prior alphas in the 2024.1 series. Same result.

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Yes

**Comment by burmancomp** [comment]
This "add or remove space before paragraph" is not announced when ctrl+0 is pressed. Obviously it should be. Result is shown in status line if showing appropriate item is enabled. I have not english office so unfortunately I do not know what item it is.

**Comment by burmancomp** [comment]
I feel there maybe other hotkeys which are not announced like shift+f3 and control+shift+a. I have word 2019 and windows 10. 

I also tried hotkeys for superscript and subscript toggles. Operation was unreliable, I mean they may or may not toggle, and nvda announcements were also unreliable.

**Comment by CyrilleB79** [comment]
> I feel there maybe other hotkeys which are not announced like shift+f3 and control+shift+a. I have word 2019 and windows 10.

See PR #10283, which is blocked. I'd probably split it up to unblock this part of it.

> I also tried hotkeys for superscript and subscript toggles. Operation was unreliable, I mean they may or may not toggle, and nvda announcements were also unreliable.

On my side, it's OK. You should open a separate issue for this one. May this be a translation problem? What is your Office language and your keyboard language? Do you switch keyboard language regularly?

**Comment by burmancomp** [comment]
Fine if more hotkeys are announced.

As to superscripts and subscripts, I think it is not translation problem. With english keylayout and when nvda language is english state changes are announced but somewhat I feel that shortcuts are not correct in microsoft own shortcut definition list [https://support.microsoft.com/en-gb/office/keyboard-shortcuts-in-word-95ef89dd-7142-4b50-afb2-f762f663ceb2](https://support.microsoft.com/en-gb/office/keyboard-shortcuts-in-word-95ef89dd-7142-4b50-afb2-f762f663ceb2). Note however that my office language is finnish so it may cause something.

As to finnish keylayout I can find shortcuts that change both states but there are problems as to announcing< shortcut which changes subscript state always is not announced.. And, there seems to be similar problem with finnish shortcut definitions [https://support.microsoft.com/fi-fi/office/wordin-pikan%C3%A4pp%C3%A4imet-95ef89dd-7142-4b50-afb2-f762f663ceb2](https://support.microsoft.com/fi-fi/office/wordin-pikan%C3%A4pp%C3%A4imet-95ef89dd-7142-4b50-afb2-f762f663ceb2).

Somewhat I think that problem might be that in nvda wrong shortcuts are used because microsoft has told wrong shortcuts.

Does those shortcuts microsoft tells work with nvda, I mean here especially superscript and subscript and shortcuts microsoft announces should work?

**Comment by CyrilleB79** [comment]
@burmancomp, please open a new issue for superscript / subscript reporting issue. I won't give any more answer in this issue to avoid hijacking it any longer. Thanks.

**Comment by CyrilleB79** [comment]
@XLTechie you write:
> The result of the toggle should be announced when pressed, like the subscript toggle (Ctrl+=).

OK. This can be done quite easily.

> NVDA+f should probably announce the state of this toggle if paragraph spacing is enabled as well.

What NVDA+f reports does not depend on if checkboxes are enabled or disabled. Moreover, there is nothing about paragraph spacing in Document formatting settings dialog. What had you in mind when writing "if paragraph spacing is enabled"?

**Comment by XLTechie** [comment]
Thanks @CyrilleB79
